### PR TITLE
fix broken tests and some fix

### PR DESCRIPTION
**Pull request by item4**
1. Fix broken tests. Cause is alembic's version up.
2. Fix subprocess issue. Some env occur bugs by wrong cwd.
3. Remove duplicated codes.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -49,6 +50,19 @@ def fx_tmpdir(tmpdir):
 
 
 @yield_fixture
+def fx_change_cwd_to_python_dir():
```
Comment (dahlia) [inline tests/conftest.py]: `change_cwd` can be shortend to `cd`.  What do you guys think about this? @clicheio/owners

Comment (item4) [inline tests/conftest.py]: hmm. I find [what is `cd`](http://ss64.com/bash/cd.html) and it mean this

> Change Directory - change the current working directory to a specific Folder.

I think `change_cwd` and `cd`, both is good. So I have no idea about this issue.

Comment (miaekim) [inline tests/conftest.py]: I think that "change current working directory to python dir" has same meaning "cd python dir". :3

**Comment by miaekim** [comment]
We can merge anyway. @item4 will you change the name of the function?

**Pull request closed without merging**